### PR TITLE
style: empty slice and redundant type conversion and .gitignore add develop tools

### PR DESCRIPTION
**Pull request by demoManito**
#### What type of PR is this?
style

#### Check the PR title.

- [x] This PR title match the format: \<type\>(optional scope): \<description\>
- [x] The description of this PR title is user-oriented and clear enough for others to understand.

#### (Optional) Translate the PR title into Chinese.
修改代码风格：
- 空切片申明方式 (https://github.com/golang/go/wiki/CodeReviewComments#declaring-empty-slices)
- 多余的类型转换
- ignore `.vscode`

#### (Optional) More detail description for this PR(en: English/zh: Chinese).

en:
zh(optional):

### CODE REVIEW

File: cmd/hz/internal/thrift/tags.go [hunk 1]
```diff
@@ -93,7 +93,7 @@ var (
 )
 
 func CheckTagOption(args *config.Argument) []generator.Option {
-	ret := []generator.Option{}
+	var ret []generator.Option
```
Comment (welkeyever) [inline cmd/hz/internal/thrift/tags.go:96]: May you give some more info or context about this change?

Comment (demoManito) [reply cmd/hz/internal/thrift/tags.go:96]: image
It's only used in one place and it doesn't affect other places

Comment (demoManito) [reply cmd/hz/internal/thrift/tags.go:96]: see: https://github.com/cloudwego/hertz/blob/b0bcce3a032cf0e450677ab8d93138f273d46492/cmd/hz/internal/generator/model.go#L89

Comment (li-jin-gou) [reply cmd/hz/internal/thrift/tags.go:96]: I think [welkeyever](https://github.com/welkeyever) wants to know what problem this PR is trying to solve, why the codes are formatted and the reason and context ?

Comment (demoManito) [reply cmd/hz/internal/thrift/tags.go:96]: https://github.com/golang/go/wiki/CodeReviewComments#declaring-empty-slices

**Review by welkeyever — APPROVED** [review]

**Review by li-jin-gou — APPROVED** [review]

**Pull request merged**